django/core/serializers/xml_serializer.py

```diff
@@ -133,36 +132,39 @@ def handle_m2m_field(self, obj, field):
         """
         if field.remote_field.through._meta.auto_created:
             self._start_relational_field(field)
-            if self.use_natural_foreign_keys and hasattr(
```

Comment:
Can you reduce this diff by keeping the original if else branches and just replacing the first if with `if use_natural`?